### DISCUSSION TITLE
Short URL

### DISCUSSION THREAD

**Discussion by kaashifhamid** [jgraph/drawio#3365] [Q&A]
Hello there,
When I publish a content, the url is too long, I can't use any url shortener (eg: bit.ly) to shortener the url. Do we have any option to generate a short url?
Thanks in advance!

**Comment by davidjgraph** [comment]
The URL contains the full data diagram data because the diagram isn't stored in a location where the data can be shared. If you store the data in Google Drive, for example, you can share the link to the Google Drive location, which is shorter.

**Reply by dvquy13** [reply]
@kaashifhamid To elaborate we can paste the Google Drive link into [this tool](https://jgraph.github.io/drawio-tools/tools/viewer.html) to generate a shorter link to our diagram viewer. Then we can further use url shortener to have an even nicer format.

**Comment by kaashifhamid** [comment]
Hello Quy,
Thanks for the information!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
